web/satellite/src/utils/constants/analyticsEventNames.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export const EVENTS = {
+    CLICKED_LOGO: 'clicked-on-logo',
+    CLICKED_LOGIN: 'clicked-on-login',
+    CLICKED_BACK_TO_LOGIN: 'clicked-on-back-to-login',
+    PROJECT_CREATED: 'Project Created',
+    PROJECT_DELETED: 'Project Deleted',
+    USER_DELETED: 'User Deleted',
+    USER_LOGGED_IN: 'User Logged In',
+    EMAIL_VERIFIED: 'Email Verified',
+    PROJECT_VIEWED: 'Project Viewed',
+    API_KEY_CREATED: 'API Key Created',
+    API_KEY_DELETED: 'API Key Deleted',
+    PAYMENT_METHODS_VIEWED: 'Payment Methods Viewed',
+    PAYMENT_METHOD_ADDED: 'Payment Method Added',
```

Comment:
nit: Sort these properties by alphabetical order, it's helpful to find them when you are developing or reviewing code.